SwiftCharts/Views/ChartAreasView.swift

```diff
@@ -51,6 +51,10 @@ open class ChartAreasView: UIView {
     }
     
     fileprivate func show(path: UIBezierPath) {
+        guard colors.count > 0 else {
```

Comment:
Thank you! Just fixed it. This way we basically only look up the array once right?